### DISCUSSION TITLE
Help With Bot Making

### DISCUSSION THREAD

**Discussion by PraseodymiumSpike** [WandererXII/lishogi#725] [Q&A] [answered]
I downloaded HaChu from Harm Muller's website and want to make a bot account using it. I can't guarantee success, but obviously, I need to know how to make a bot before I can even start.

**Comment by M-DinhHoangViet** [comment, marked as answer]
https://github.com/TheYoBots/Lishogi-Bot

**Comment by TheYoBots** [comment]
https://github.com/TheYoBots/Lishogi-Bot is the way to go when making a lishogi bot, but HaChu is an engine that runs on the xboard/winboard protocol and [Lishogi-Bot](https://github.com/TheYoBots/Lishogi-Bot) only supports USI engines. So to run HaChu there is currently no way.
